src/model/SizingSystem.cpp

```diff
@@ -1632,15 +1632,6 @@ namespace model {
     return getImpl<detail::SizingSystem_Impl>()->autosizedOccupantDiversity();
   }
 
-  // DEPRECATED: TODO REMOVE as soon as standards > 0.29.0 is relased
-  boost::optional<double> SizingSystem::minimumSystemAirFlowRatio() const {
-    return getImpl<detail::SizingSystem_Impl>()->centralHeatingMaximumSystemAirFlowRatio();
-  }
-  // DEPRECATED: TODO REMOVE as soon as standards > 0.29.0 is relased
-  bool SizingSystem::setMinimumSystemAirFlowRatio(double centralHeatingMaximumSystemAirFlowRatio) {
-    return getImpl<detail::SizingSystem_Impl>()->setCentralHeatingMaximumSystemAirFlowRatio(centralHeatingMaximumSystemAirFlowRatio);
-  }
```

Comment:
OpenStudio-resources needs adjustment: I'd want a PR that fixes stuff and confirmation that everything still runs.

this one is disabled but should still be fixed: https://github.com/NREL/OpenStudio-resources/blob/6c90661c95c0e896ac447eefaa153c9205319534/model/simulationtests/afn_single_zone_ac.rb#L56